arrays_assignment.js

```diff
@@ -0,0 +1,292 @@
+// // # Arrays Lab
+// //
+// // To complete this lab (as for most labs), do the following:
+// //
+// // 1. Fork this repo.
+// // 2. Clone **your** forked repo.
+// // 3. Create a file called `arrays_lab.js` inside your cloned repo.
+// // 4. Copy the description of the exercise as a comment.
+// // 5. Underneath the commented description write your code.
+// // 6. Ensure your solutions work properly!
+// // 7. Open PR against this repo.
+// //
+// // ___
+// //
+// // ## Exercises
+// //
+//
+// let myArray = [1,2,3,4,5,6,7,8,9];
+// // 1. Log the last element of an array.
+// console.log(myArray.length);
+//
+// // 2. Change the first element of an array to the string "something else".
+// myArray[0] = "something else";
+// console.log(myArray[0]);
+// // 3. Add "mango" to the end of a fruits array ["strawberry", "banana"].
+// let fruitsArray = ["strawberry", "banana"];
+// fruitsArray.push("mango");
+// console.log(fruitsArray);
+// // 4. Add "blueberry" to the front of the same fruits array.
+// fruitsArray.unshift("blueberry");
+// console.log(fruitsArray);
+// // 5. Remove the last element of the fruits array.
+// fruitsArray.pop();
+// console.log(fruitsArray);
+// // 6. console.log the length of the fruits array.
+// console.log(fruitsArray.length);
+// // 7. Remove the first element of the fruits array.
+// fruitsArray.shift();
+// console.log(fruitsArray);
+// // 8. Join all the elements in the fruits array with a '$'.
+// fruitslist = fruitsArray.join("$");
+// console.log(fruitslist);
+//
+// //
+// //
+// // 9. Given two arrays, write code that logs the larger array.  If the arrays are the same length, log "They are the same size"
+// //
+// // ```js
+// // let firstArr = [1,2,3]
+// // let secondArr = [1,5,2,4]
+// //
+// // // log [1,5,2,4]
+// // ```
+// let array1 = [1,2,3,4,5];
+// let array2 = [1,2,4,5,6,7,8,9];
+//
+// let arr_length1 = array1.length;
+// let arr_length2 = array2.length;
+//
+// arr_length1 > arr_length2 ? console.log(array1) : console.log(array2);
+
+// 10. Given an array, log its middle element.  If the middle of the array is between two elements, log both of them.
+//
+// ```js
+// let hasExactMiddleArr = [1,5,3]
+//
+// // log 5
+//
+// let hasNoExactMiddleArr = [1,4,6,9]
+//
+// // log 4,6
+//
+// ```
+// let hasExactMiddleArr = [1,2,3];
+// let middle = hasExactMiddleArr.length/2;
+// console.log(hasExactMiddleArr.length);
+// console.log(middle);
+// console.log(hasExactMiddleArr[Math.floor(middle)]);
+
+ let hasNoExactMiddleArr = [1,4,6,9];
```

Comment:
Using if statement to determine which output to log would've been better. Ex: 
if(arr.length % 2 !== 0) {
console.log(arr[middle])
} else {
console.log(arr[middle], arr[middle - 1])
}